library/src/main/java/com/chuckerteam/chucker/internal/support/ClearDatabaseService.kt

```diff
@@ -9,7 +9,7 @@ internal class ClearDatabaseService : IntentService(CLEAN_DATABASE_SERVICE_NAME)
 
     override fun onHandleIntent(intent: Intent?) {
         System.err.println("onHandleIntent")
```

Comment:
I believe that it is not correct to use `err` here, since it is called from normal action, not error. Suggest to replace it or remove.